redash/query_runner/drill.py

```diff
@@ -0,0 +1,179 @@
+import json
+import logging
+import sys
+import os
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+logger = logging.getLogger(__name__)
+
+try:
+    import pyodbc
+    enabled = True
+    logger.warning("LD_LIBRARY_PATH: {}".format(os.environ.get('LD_LIBRARY_PATH')))
+    logger.warning("ODBCINI: {}".format(os.environ.get('ODBCINI')))
+    logger.warning("MAPRDRILLINI: {}".format(os.environ.get('MAPRDRILLINI')))
+except ImportError, e:
+    logger.warning("Missing dependencies. Please install pyodbc.")
+    logger.warning("You can use pip: pip install pyodbc")
+    enabled = False
+
+COLUMN_NAME = 0
+COLUMN_TYPE = 1
+
+types_map = {
+    'BIGINT': TYPE_INTEGER,
+    'INTEGER': TYPE_INTEGER,
+    'INT': TYPE_INTEGER,
+    'DOUBLE': TYPE_FLOAT,
+    'long': TYPE_INTEGER,
+    'DECIMAL': TYPE_FLOAT,
+    'DEC': TYPE_FLOAT,
+    'NUMERIC': TYPE_FLOAT,
+    'FLOAT': TYPE_FLOAT,
+    'REAL': TYPE_FLOAT,
+    'BOOLEAN': TYPE_BOOLEAN,
+    'bool': TYPE_BOOLEAN,
+    'TIMESTAMP': TYPE_DATETIME,
+    'TIME': TYPE_DATETIME,
+    'DATE': TYPE_DATETIME,
+    'CHAR': TYPE_STRING,
+    'STRING': TYPE_STRING,
+    'str': TYPE_STRING,
+    'VARCHAR': TYPE_STRING,
+    'CHARACTER': TYPE_STRING,
+    'CHARACTER VARYING': TYPE_STRING,
+    # Unsure about the interval type
+    'INTERVAL': TYPE_DATETIME,
+}
+
+
+class Drill(BaseQueryRunner):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "ConnectionType": {
```

Comment:
Please use consistent naming for the configuration properties. Preferably snake cases, i.e.: `connection_type`.

Also provide a `title` to give a friendly name, specially for the ZK ones.